lambda-notes/src/reducers/notesReducer.js

```diff
@@ -0,0 +1,119 @@
+
+import {
+  FETCH_NOTES,
+  FETCH_NOTES_SUCCESS,
+  FETCH_NOTES_FAILURE,
+  DELETE_NOTE,
+  DELETE_NOTE_SUCCESS,
+  DELETE_NOTE_FAILURE,
+  POST_NOTE,
+  POST_NOTE_SUCCESS,
+  POST_NOTE_FAILURE,
+  PUT_NOTE_SUCCESS,
+  PUT_NOTE,
+  PUT_NOTE_FAILURE,
```

Comment:
ahhh so scary. So many imports...
```suggestion
import * as aliasName from '../actions'
after can access actionTypes like :
aliasName.FETCH_NOTES:
```